Prog/BPK Summary Tabs_2014-18.sas

```diff
@@ -0,0 +1,1493 @@
+			PctMrtgorigBlk_2017 = nummrtgorigblack_2017/nummrtgorigwithrace_2017;
+			PctMrtgorigBlk_2018 = nummrtgorigblack_2018/nummrtgorigwithrace_2018;
+				
+			PctMrtgorigAsn_2009 = nummrtgorigasian_2009/nummrtgorigwithrace_2009;
+			PctMrtgorigAsn_2010 = nummrtgorigasian_2010/nummrtgorigwithrace_2010;
+			PctMrtgorigAsn_2011 = nummrtgorigasian_2011/nummrtgorigwithrace_2011;
+			PctMrtgorigAsn_2012 = nummrtgorigasian_2012/nummrtgorigwithrace_2012;
+			PctMrtgorigAsn_2013 = nummrtgorigasian_2013/nummrtgorigwithrace_2013;
+			PctMrtgorigAsn_2014 = nummrtgorigasian_2014/nummrtgorigwithrace_2014;
+			PctMrtgorigAsn_2015 = nummrtgorigasian_2015/nummrtgorigwithrace_2015;
+			PctMrtgorigAsn_2016 = nummrtgorigasian_2016/nummrtgorigwithrace_2016;
+			PctMrtgorigAsn_2017 = nummrtgorigasian_2017/nummrtgorigwithrace_2017;
+			PctMrtgorigAsn_2018 = nummrtgorigasian_2018/nummrtgorigwithrace_2018;
+				
+			PctMrtgorigHisp_2009 = nummrtgorighisp_2009/nummrtgorigwithrace_2009;
+			PctMrtgorigHisp_2010 = nummrtgorighisp_2010/nummrtgorigwithrace_2010;
+			PctMrtgorigHisp_2011 = nummrtgorighisp_2011/nummrtgorigwithrace_2011;
+			PctMrtgorigHisp_2012 = nummrtgorighisp_2012/nummrtgorigwithrace_2012;
+			PctMrtgorigHisp_2013 = nummrtgorighisp_2013/nummrtgorigwithrace_2013;
+			PctMrtgorigHisp_2014 = nummrtgorighisp_2014/nummrtgorigwithrace_2014;
+			PctMrtgorigHisp_2015 = nummrtgorighisp_2015/nummrtgorigwithrace_2015;
+			PctMrtgorigHisp_2016 = nummrtgorighisp_2016/nummrtgorigwithrace_2016;
+			PctMrtgorigHisp_2017 = nummrtgorighisp_2017/nummrtgorigwithrace_2017;
+			PctMrtgorigHisp_2018 = nummrtgorighisp_2018/nummrtgorigwithrace_2018;
+				
+			PctMrtgorigOth_2009 = nummrtgorigotherx_2009/nummrtgorigwithrace_2009;
+			PctMrtgorigOth_2010 = nummrtgorigotherx_2010/nummrtgorigwithrace_2010;
+			PctMrtgorigOth_2011 = nummrtgorigotherx_2011/nummrtgorigwithrace_2011;
+			PctMrtgorigOth_2012 = nummrtgorigotherx_2012/nummrtgorigwithrace_2012;
+			PctMrtgorigOth_2013 = nummrtgorigotherx_2013/nummrtgorigwithrace_2013;
+			PctMrtgorigOth_2014 = nummrtgorigotherx_2014/nummrtgorigwithrace_2014;
+			PctMrtgorigOth_2015 = nummrtgorigotherx_2015/nummrtgorigwithrace_2015;
+			PctMrtgorigOth_2016 = nummrtgorigotherx_2016/nummrtgorigwithrace_2016;
+			PctMrtgorigOth_2017 = nummrtgorigotherx_2017/nummrtgorigwithrace_2017;
+			PctMrtgorigOth_2018 = nummrtgorigotherx_2018/nummrtgorigwithrace_2018;
+				
+			/*Births*/
+			
+			PctBirthsPrenatInad_2003 = births_prenat_inad_2003/births_w_prenat_2003; 
+			PctBirthsPrenatInad_2004 = births_prenat_inad_2004/births_w_prenat_2004; 
+			PctBirthsPrenatInad_2005 = births_prenat_inad_2005/births_w_prenat_2005; 
+			PctBirthsPrenatInad_2006 = births_prenat_inad_2006/births_w_prenat_2006; 
+			PctBirthsPrenatInad_2007 = births_prenat_inad_2007/births_w_prenat_2007; 
+			PctBirthsPrenatInad_2008 = births_prenat_inad_2008/births_w_prenat_2008; 
+			PctBirthsPrenatInad_2009 = births_prenat_inad_2009/births_w_prenat_2009; 
```

Comment:
Just noting there is a gap in the birth data in 2009. You will see missing data for this year.